admin-frontend/.neutrinorc.js

```diff
@@ -1,41 +1,51 @@
-const merge = require('babel-merge');
-const path = require('path');
-const react = require('@neutrinojs/react');
+const merge = require("babel-merge");
+const path = require("path");
+const react = require("@neutrinojs/react");
 
 module.exports = {
   options: {
-    root: __dirname
+    root: __dirname,
   },
   use: [
     react({
       html: {
-        title: 'Admin Panel',
-        favicon: './src/assets/img/favicon.ico'
+        title: "Admin Panel",
+        favicon: "./src/assets/img/favicon.ico",
       },
       devServer: {
         proxy: {
-          '/api': 'http://localhost:5000'
-        }
-      }
+          "/api": "http://localhost:5000",
+        },
+      },
     }),
     neutrino => {
       neutrino.config.performance.hints(false);
 
-      neutrino.config.output
-        .publicPath('');
+      neutrino.config.output.publicPath("");
 
       neutrino.config.module //needed because of how neutrino handles plugin configuration
-        .rule('compile')
-        .use('babel')
-        .tap(options => merge({
-          plugins: [
-            [require.resolve('@babel/plugin-proposal-decorators'), {legacy: true}],
-            [require.resolve('@babel/plugin-proposal-class-properties'), {loose: true}],
-            require.resolve('@babel/plugin-transform-classes'),
-            [require.resolve('@babel/plugin-transform-regenerator'), {asyncGenerators: false}],
-            require.resolve('@babel/plugin-transform-runtime')
-          ]
-        }, options));
-    }
-  ]
+        .rule("compile")
+        .use("babel")
+        .tap(options =>
+          merge(
+            {
+              plugins: [
+                [
+                  require.resolve("@babel/plugin-proposal-decorators"),
+                  { legacy: true },
+                ],
+                [
+                  require.resolve("@babel/plugin-proposal-class-properties"),
+                  { loose: true },
+                ],
+                require.resolve("@babel/plugin-transform-classes"),
+                //[require.resolve('@babel/plugin-transform-regenerator'), {asyncGenerators: false}],
```

Comment:
Are we using function generator anywhere in the codebase at all? If it's not beeded, we can just remove the commented out code.